src/main/java/soconnect/ui/PersonCard.java

```diff
@@ -58,73 +59,92 @@ public PersonCard(Person person, int displayedIndex, Logic logic) {
+            flowpanes[1].getChildren().forEach(label -> label.setStyle(
+                    "-fx-font-size: 12;-fx-font-family: \"Segoe UI Semibold\";"));
         }
         if (flowpanes[2] != null) {
             flowpanes[2].getChildren().add(new Label(person.getPhone().value));
+            flowpanes[2].getChildren().forEach(label -> label.setStyle(
+                    "-fx-font-size: 12;-fx-font-family: \"Segoe UI Semibold\";"));
         }
         if (flowpanes[3] != null) {
             person.getTags().stream()
                     .sorted(Comparator.comparing(tag -> tag.tagName))
-                    .forEach(tag -> flowpanes[3].getChildren().add(new Label(tag.tagName)));
+                    .forEach(tag -> flowpanes[3].getChildren().add((new Label(tag.tagName))));
+            flowpanes[3].getChildren().forEach(label -> label.setStyle("-fx-background-color: #3e7b91;"
+                    + "-fx-font-size: 11;-fx-background-radius: 2;"
+                    + "-fx-border-radius: 2;-fx-padding: 1 3 1 3;"));
         }
     }
 
     /**
-     * Filters the attributes based on what attributes were chosen to be hidden.
+     * Generates the order of the attributes based on the order set by the user.
+     *
+     * @return The required order.
+     */
+    private int[] orderAttributes() {
+        int[] order = new int[NUMBER_OF_CUSTOMISABLE_ATTRIBUTES];
+        String orderStr = logic.getAttributeOrder();
+        String[] orderArr = orderStr.trim().split(">");
+
+        if (orderArr.length != NUMBER_OF_CUSTOMISABLE_ATTRIBUTES) {
+            //Returns default order when the orderStr is not in correct format.
+            order = new int[]{3, 2, 1, 0};
+            return order;
+        }
+
+        try {
+            for (int i = 0; i < NUMBER_OF_CUSTOMISABLE_ATTRIBUTES; i++) {
+                order[i] = convertToIndex(orderArr[i]);
+            }
+        } catch (IllegalArgumentException e) {
+            order = new int[]{3, 2, 1, 0};
+        }
+
+        return order;
+    }
+
+    /**
+     * Filters the attributes based on what attributes were chosen to be hidden by setting the hidden attributes'
+     * associated index to the index of the empty last FlowPane element.
      *
      * @param order The order that is unfiltered.
-     * @return The order with attributes filtered where 4 represents a filtered attribute.
+     * @return The order with attributes filtered.
      */
     private int[] filterAttributes(int[] order) {
-        boolean[] isHidden = new boolean[4];
-        GuiSettings currSettings = logic.getGuiSettings();
-        String currHiddenAttributes = currSettings.getHiddenAttributes().trim();
+        boolean[] isHidden = new boolean[NUMBER_OF_CUSTOMISABLE_ATTRIBUTES];
+        String currHiddenAttributes = logic.getHiddenAttributes();
 
         if (!currHiddenAttributes.equals("NONE")) {
-            String[] strArr = currHiddenAttributes.split(",");
```

Comment:
Might be better to declare the `"NONE"` as a constant somewhere, since it is used in quite a lot of places.

Reply:
OK I shall do that.